examples-chart/templates/tests/event-subscription/lambda/Pod.yaml

```diff
@@ -0,0 +1,32 @@
+{{ if .Values.examples.eventSubscription.lambda.deploy }}
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Release.Name }}-event-subscription-lambda-test-{{ randAlphaNum 5 | lower }}"
+  labels:
+    example: event-subscription-lambda
+    chart: examples-tests
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  serviceAccountName: {{ .Release.Name }}-examples-service-account
+  containers:
+    - name: event-subscription-lambda-test
+      image: {{ .Values.examples.image }}
+      command: ['sh', '-c']
+      args: [' 
+        export KYMA_EXAMPLE_ENV="{{ .Release.Namespace }}" &&
+        cd event-subscription/lambda &&
+        kubectl apply -f deployment/sample-publisher.yaml -n $KYMA_EXAMPLE_ENV &&
+        sleep 10 &&
+        kubectl exec -n $KYMA_EXAMPLE_ENV $(kubectl get pods -n $KYMA_EXAMPLE_ENV -l app=sample-publisher --output=jsonpath={.items..metadata.name}) -c sample-publisher -- curl -i -H "Content-Type: application/json" -X POST http://core-publish.kyma-system:8080/v1/events -d ''{"source-id": "stage.commerce.kyma.local", "event-type": "hello", "event-type-version": "v1", "event-time": "2018-11-02T22:08:41+00:00", "data": { "order-number": 123 }}''
```

Comment:
I believe you are aware you only test the publish part. Not the lambda triggering part for example.